vm/systemSmartContracts/governance.go

```diff
@@ -579,7 +582,13 @@ func (g *governanceContract) updateUserVoteListV2(address []byte, nonce uint64,
 		return err
 	}
 
-	return g.saveUserVotesV2(address, userVoteList)
+	err = g.saveUserVotesV2(address, userVoteList)
```

Comment:
In this function, we do a get -> update -> save -> get (for clear) -> save (after clear). I think we can optimize this by making the `clearUserVotes` do actually one thing: clear the old data for a provided userVoteList and move the call before the last save.

Reply:
yep. no need to save 2 times. optimise this.

Reply:
fixed